packages/frontend/src/pages/Account.tsx

```
@@ -0,0 +1,149 @@
+  //   if (!isAuthenticated) {
+  //     return null;
+  //   }
+
+  // Optionnel : restriction par rôle
+  if (!isAuthenticated || (user?.role !== 'user' && user?.role !== 'admin')) {
+    return (
+      <div className="flex flex-col justify-center items-center container mx-auto px-4 min-h-screen text-center py-12">
+        <Lock className="w-10 h-10 mb-4" />
+        <h1 className="text-3xl font-bold mb-8">Accès refusé</h1>
+        <Button asChild>
+          <Link to="/">Retour à l'accueil</Link>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)